file: test cases fail

Seems the build server does not build the PRs anymore. Anyway, broken test cases slipped through.

```
 71/170 Test  #71: testmod_file .......................***Failed    0.00 sec
FILE     TESTS
==================

/home/markus/Projekte/Elektra/current/src/plugins/file/testmod_file.c:27: error in testReadSingleLine: call to kdbGet was not successful
/home/markus/Projekte/Elektra/current/src/plugins/file/testmod_file.c:30: fatal in testReadSingleLine: key not found

```

--- a/src/plugins/file/file/multiline
+++ b/src/plugins/file/file/multiline
@@ -1,0 +1,7 @@
+
+this
+
+	is a
+   multi line test-
+file
+

--- a/src/plugins/file/file/singleline
+++ b/src/plugins/file/file/singleline
@@ -1,0 +1,1 @@
+this is a single line testfile

